R/data.R

```diff
@@ -0,0 +1,465 @@
+#' @format ## `hg38_lymphoma_genes_bed`
+#' A data frame with 195 rows and 4 columns.
+#' \describe{
+#'   \item{chromosome_name}{The chromosome for which the gene is residing on}
+#'   \item{start_position}{The start coordinate for the gene}
+#'   \item{end_position}{The end coordinate for the gene}
+#'   \item{hgnc_symbol}{Gene symbol in Hugo format}
+#' }
+"hg38_lymphoma_genes_bed"
+
+
+#' hg38 Partner Genes.
+#'
+#' Translocation partners for oncogenes in with coordinates in respect to hg38.
+#'
+#' @format ## `hg38_partners`
+#' A data frame with 31 rows and 5 columns.
+#' \describe{
+#'   \item{chrom}{The chromsome for which the gene is residing on}
+#'   \item{start}{The start coordinate for the gene}
+#'   \item{end}{The end coordinate for the gene}
+#'   \item{gene}{Gene symbol in Hugo format}
+#'   \item{entrez}{Entrez ID}
+#' }
+"hg38_partners"
+
+
+#' grch37 Hotspot Regions.
+#'
+#' Mutation hotspot regions in respect to grch37.
+#'
+#' @format ## `hotspot_regions_grch37`
+#' A data frame with 6 rows and 3 columns.
+#' \describe{
+#'   \item{chrom}{Chromosome for the described region}
+#'   \item{start}{The start coordinate for the region}
+#'   \item{end}{The end coordinate for the region}
+#' }
+"hotspot_regions_grch37"
+
+
+#' hg38 Hotspot Regions.
+#'
+#' Mutation hotspot regions in respect to hg38.
+#'
+#' @format ## `hotspot_regions_hg38`
+#' A data frame with 6 rows and 3 columns.
+#' \describe{
+#'   \item{chrom}{Chromosome for the described region}
+#'   \item{start}{The start coordinate for the region}
+#'   \item{end}{The end coordinate for the region}
+#' }
+"hotspot_regions_hg38"
+
+
+#' Lymphoma Genes Comprehensive.
+#'
+#' A detailed data frame with lymphoma genes, annotated with evidence from literature and aSHM. 
+#'
+#' @format ## `lymphoma_genes_comprehensive`
+#' A data frame with 127 rows and 9 columns.
+#' \describe{
+#'   \item{ensemble_gene_id}{Gene in Ensemble format}
+#'   \item{Gene}{Gene symbol in Hugo format.}
+#'   \item{Chapuy}{Boolean flag, TRUE if gene verified by the stated study (Chapuy)}
+#'   \item{Reddy}{Boolean flag, TRUE if gene verified by the stated study (Reddy)}
+#'   \item{LymphGen}{Boolean flag, TRUE if lymphGen}
+#'   \item{curated}{Boolean flag, describing if the gene has been curated or not}
+#'   \item{other}{Other support for the described gene}
+#'   \item{Lacy}{Boolean flag, TRUE if gene verified by the stated study (Lacy)}
+#'   \item{aSHM}{Boolean flag for annotating aSHM}
+#' }
+"lymphoma_genes_comprehensive"
+
+
+#' Lymphoma Genes
+#'
+#' A data frame with known lymphoma genes, genes are annotated by pathology, as well as literature support.
+#'
+#' @format ## `lymphoma_genes`
+#' A data frame with 196 rows and 12 columns.
+#' \describe{
+#'   \item{Gene}{Gene symbol}
+#'   \item{DLBCL}{Boolean flag annotating id the described genes are significant for the pathology (DLBCL)}
+#'   \item{FL}{Boolean flag annotating id the described genes are significant for the pathology (FL)}
```

Comment:
Probably `id` swap for `if` in these lines?

Reply:
Oh, yes, indeed!